### ISSUE TITLE
Overlapped between model training set and test set

### ISSUE THREAD

**Issue by kpinto-gil** [phi-grib/flameWeb#10]
It would be very useful to have a Venn diagram showing the overlapping between two models selected, my model and my test set, ...
